.github/workflows/solvcon_runner.yml

```diff
@@ -0,0 +1,79 @@
+name: Application Build - SOLVCON
+on:
+  schedule:
+    - cron: '21 18 * * 5'
```

Comment:
Run weekly. The content has been verified with my own Action run. Let's wait for the next Friday to see if the run is launched expectedly.